kafka-ui-react-app/src/components/Topics/List/TopicTable.tsx

```diff
@@ -8,11 +8,57 @@ import { useSearchParams } from 'react-router-dom';
 import ClusterContext from 'components/contexts/ClusterContext';
 import { useTopics } from 'lib/hooks/api/topics';
 import { PER_PAGE } from 'lib/constants';
+import BytesFormatted from 'components/common/BytesFormatted/BytesFormatted';
 
 import { TopicTitleCell } from './TopicTitleCell';
 import ActionsCell from './ActionsCell';
 import BatchActionsbar from './BatchActionsBar';
 
+function formatThroughput(row: Topic) {
+  const production = row.bytesInPerSec;
+  const consumption = row.bytesOutPerSec;
+  if (production === undefined && consumption === undefined) {
+    return (
+      <tr>
+        <td>N/A</td>
+      </tr>
+    );
+  }
+  if (production === undefined) {
+    return (
+      <tr>
+        <td>
+          out: <BytesFormatted value={consumption} />
+        </td>
+      </tr>
+    );
+  }
+  if (consumption === undefined) {
+    return (
+      <tr>
+        <td>
+          in: <BytesFormatted value={production} />
+        </td>
+      </tr>
+    );
+  }
+
+  return (
+    <div>
+      <tr>
+        <td>
+          in: <BytesFormatted value={production} />
+        </td>
+      </tr>
+      <tr>
+        <td>
+          out: <BytesFormatted value={consumption} />
+        </td>
+      </tr>
+    </div>
```

Comment:
Sorry, not sure I understand how this html construction should work in a table cell. And please let's avoid wrapping `tr` to `div`.

Reply:
Can do. What's the best way to let the text span more than one line? That was specified in the issue.

Reply:
Seems to work without using `tr' and `td`.